app/views/views.py

```diff
@@ -57,15 +69,24 @@ def __init__(self):
                                    location='json')
         super(Question, self).__init__()
 
-    def get(self, qtn_id):
+    def get(self, Question_ID):
         question = [
-            question.__dict__ for question in Questions if question.qtn_id == qtn_id]
+            question.__dict__ for question in Questions if question.Question_ID == Question_ID]
         if len(question) == 0:
             return make_response(jsonify(
                 {'message': 'Sorry no questions asked yet'}
             ))
         return make_response(jsonify(question), 200)
 
+    def delete(self, Question_ID):
+        """Method for Deleting a Question"""
+        delete_qtn = [
+            qtn for qtn in Questions if qtn.Question_ID == Question_ID]
+        if len(delete_qtn) == 0:
+            abort(404)
+        Questions.remove(delete_qtn[0])
```

Comment:
Since your question ID's are utilising the length of the list, I would expect that at this point, as you delete questions, the remaining ID's are updated. Otherwise you will have repeated ID's and missing ID's as the data is added and removed.